### PR TITLE
snackbar, input components created

### PR DESCRIPTION
**Pull request by agrawalshanu708**
made a snackbar & input components.
snackbar of add to cart, Email
made a sign-up form. login form , radio & checkbox
link: https://nervous-mcclintock-a6b287.netlify.app/

### CODE REVIEW

**Comment by agrawalshanu708** [comment]
made a snackbar & input components.
snackbar of add to cart, Email
made a sign-up form. login form , radio & checkbox
link: https://nervous-mcclintock-a6b287.netlify.app/

**Review by tanishk15sharma — APPROVED** [review]
good to merge

File: input/input.css [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+@media screen and (max-width: 768px) {
+
+
+}
+
+
+
+/* ------------------------------------------------------------------------------------------------------------ */
+.skin{
+    padding: 0.5rem 0.5rem 0 0.1rem;
```
Comment (shubhambishnoi) [inline input/input.css:13]: don't convert values less than 10 px to rems as that will further reduce the size on smaller screens as suggested by Vikram

**Review by shubhambishnoi — APPROVED** [review]

File: input/input.css (hunk 1 above)
Comment (agrawalshanu708) [reply input/input.css:13]: ohkay i ll see that ,thanx

File: input/input.css [hunk 2]
```diff
@@ -0,0 +1,102 @@
+@import "../resetcss.css";
+
+
+@media screen and (max-width: 768px) {
+
```
Comment (zabihshaik) [inline input/input.css:5]: If there is no code, can delete these lines.

File: input/input.css [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+      gap: 1rem;
+      /* margin : 2rem 0 8rem 0; */
+  }
+  
+
+li{
+    font-size: 1.8em;
+    margin: 0.5rem
+    }
+    .list-skin{
```
Comment (zabihshaik) [inline input/input.css:46]: Better to use Prettier to format Code to look good.

File: input/input.html [hunk 4]
```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (zabihshaik) [inline input/input.html:6]: Better to add meta tag of name and description

**Review by zabihshaik — COMMENTED** [review]
Everything looks good. But, a few suggestions:

- Better to remove unnecessary blank spaces. And, if If there is no code, can delete those lines.(like the one's in a media query).
- Better to use Prettier to format the Code to look good and easy to review.
- Better to add meta tag of name and description in the head of html files.
- Regarding the folder structure, it will be good to have all html files in a folder, eg. pages and all the css files in a folder, eg. styles and all the JavaScript files in JS folder. 
- It would be better if screenshots of changes( Components) are added in Description.

Appreciate the way you have implemented Snackbar JS functionality.

**Comment by agrawalshanu708** [comment]
thanks zabihshaik for your valuable suggetion

**Pull request merged**